### DISCUSSION TITLE
Always getting two 2FA SMS

### DISCUSSION THREAD

**Discussion by kagura114** [community/community#64621] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Since last week or some time earlier, I found when I was trying to get 2fa code via SMS, I got two codes from different senders and have different marks. Most of the time they're same but in some cases I got two different codes and only one of them was valid. This problem never happened before and it doesn't stop me from logging in.
Also I have tested on multiple devices and it is always happening.

**Comment by arafajs** [comment]
It is possible that you are receiving two different 2FA codes from different senders because the service provider is using two different providers for SMS delivery. This is not uncommon, as different providers have different pricing and features.

The different marks on the codes are likely just a way for the providers to distinguish their codes from each other. For example, one provider might use a hashtag (#) and the other provider might use an asterisk (*).

If you are concerned about the security of your 2FA codes, you can try using an authenticator app instead of SMS. Authenticator apps generate 2FA codes locally on your device, so they are less vulnerable to interception.

Here are some other things you can do to protect your 2FA codes:

- Make sure that your phone number is up to date with the service provider.
- Keep your phone's operating system up to date.
- Enable two-factor authentication for your email account.
- Be careful about clicking on links in emails or text messages.

If you are still concerned about the security of your 2FA codes, you can contact the service provider for more information.

Here are some additional things to consider:

- The sender ID for the second code might be different because it is coming from a different country or region.
- The second code might be a backup code that you generated in case you lose your phone.
- It is also possible that someone is trying to hack your account and is sending you fake 2FA codes. If you are concerned about this, you can change your password and enable two-factor authentication for your account.
